contracts/src/test/kotlin/com/template/contracts/ContractTests.kt

```diff
@@ -13,24 +14,55 @@ class ContractTests {
     private val ledgerServices: MockServices = MockServices(listOf("com.template"),
         TestIdentity(CordaX500Name("Test","Test","US")),
         networkParameters = testNetworkParameters(minimumPlatformVersion = 4))
-    var doctor = TestIdentity(CordaX500Name("Alice", "TestLand", "US"))
+
+    var doctor = TestIdentity(CordaX500Name("Doc", "TestLand", "US"))
     var alice = TestIdentity(CordaX500Name("Alice", "TestLand", "US"))
-    var bob = TestIdentity(CordaX500Name("Alice", "TestLand", "US"))
+    var bob = TestIdentity(CordaX500Name("Bob", "TestLand", "US"))
+
+    val appointmentReferenceState = AvailableAppointmentDate("16-01-2000", doctor.party, alice.party, bob.party)
+    val requestState = AppointmentRequest("16-01-2000", doctor.party, alice.party)
+    val appointmentState = Appointment("16-01-2000", doctor.party, alice.party)
 
     @Test
     fun requestTest() {
```

Comment:
I like to check signers in contract tests - for example if the transaction fails if Alice creates the date, or if Bob can approve/deny a request.